test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -137,19 +139,27 @@ public SignatureTrustAndValidityVerificationProviderTests(SigningTestFixture fix
                 LogLevel.Warning,
                 setting);
 
-            Assert.Empty(matchingIssues);
+            Assert.Equal(2, matchingIssues.Count);
```

Comment:
I spoke to @PatoBeltran about this specific scenario. I thought we wanted restore to be silent in this case.

Reply:
Yes, if a restore is ran from an offline scenario we shouldn't fill the user with a warning per package telling him revocation information is unavailable.